src/main/webapp/resources/js/components/samples/cartSamplesSlice.ts

```diff
@@ -55,7 +56,8 @@ const cartSamplesSlice = createSlice({
 
     builder.addCase(removeCartSampleId, (state, action) => {
       state.sampleIds = state.sampleIds.filter(
-        (sampleId) => sampleId !== parseInt(action.payload.sampleId)
+        (sampleId: string | number) =>
```

Comment:
Both types here again?

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f